### ISSUE TITLE
Create AWS AMI for 0.4.3

### ISSUE THREAD

**Issue by mikelorant** [rancher/os#811]
The AWS AMI's aren't updated for the 0.4.3 release. This means that Docker 1.10.x isn't available out of the box. While it is possible to use ros os upgrade, this significantly slows down the setup time.

Also, clarify the cloud-config when using:
```
  upgrade:
    url: https://releases.rancher.com/os/releases.yml
    image: rancher/os
```
Does this force an update on first boot? Do I need to set the image to rancher/os:v0.4.3?

**Comment by SydOps** [comment]
+1

**Comment by ryansch** [comment]
You can try building your own: https://forums.rancher.com/t/how-to-build-rancheros-amis/1672

**Comment by SydOps** [comment]
@ryansch

did you know the packer way to build the ami image: https://github.com/rancher/os-packer

**Comment by ryansch** [comment]
@SydOps That's quite old and hasn't been updated since 0.3.0.

**Comment by deniseschannon** [comment]
Published the v0.4.3 AMIs.

**Comment by SydOps** [comment]
Thanks @denischannon. could you please update the packer reopsitory and share us the template to us as well?
